docs/cli.md

```diff
@@ -118,6 +118,19 @@ Set the current working directory.
 zx --cwd=/foo/bar script.mjs
 ```
 
+## --env
+Specify a env file.
+
+```bash
+zx --env=/path/to/some.env script.mjs
+```
+
+Also works with `--cwd` options.
```

Comment:
When `cwd` option is specified, it will used as base path: `--cwd='/foo/bar' --env='../.env'` → '/foo/.env'